server/container/cli/src/main/java/org/apache/james/cli/probe/impl/JmxSieveProbe.java

```diff
@@ -72,7 +78,14 @@ public void removeSieveQuota(String user) throws Exception {
     }
 
     @Override
-    public void addActiveSieveScript(String user, String name, String script) throws Exception {
+    public void addActiveSieveScript(String user, String toFileName, String fromScriptFileName) throws Exception {
+        sieveRepositoryManagement.addActiveSieveScript(user, sanitizeScriptName(toFileName),
+                FileUtils.readFileToString(new File(fromScriptFileName), StandardCharsets.UTF_8));
```

Comment:
maybe we could ensure we won't break the server with a too large script by using a LimitedInputStream ?

Reply:
ManageSieve does not seem to limit script size (I could not find any. I might have missed it). This would make the new functions inconsistent with existing fuctionality.

Reply:
you are right about that, I just opened https://issues.apache.org/jira/browse/JAMES-2190